app/models/product.rb

```diff
@@ -0,0 +1,2 @@
+class Product < ApplicationRecord
+end
```

Comment:
I would add some validations, a product should probably not have any empty attributes?